commands/daemon.go

```diff
@@ -96,6 +97,12 @@ func daemonRun(req *cmds.Request, re cmds.ResponseEmitter) error {
 	opts = append(opts, node.BlockTime(blockTime))
 	opts = append(opts, node.ClockConfigOption(clock.NewSystemClock()))
 
+	journal, err := journal.NewZapJournal(rep.JournalPath())
+	if err != nil {
+		return err
+	}
+	opts = append(opts, node.JournalConfigOption(journal))
+
```

Comment:
Only create a zap backed journal when something runs the daemon via the `daemon` command.